pythonosc/osc_tcp_server.py

```diff
@@ -0,0 +1,392 @@
+# TODO: timeouts!
```

Comment:
please remove the comment and leave an open issue for anyone willing to take it instead, thank you :)

Reply:
This was an artifact from the earlier implementation - I don't know exactly what it means to I removed it.